moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryInterimID.js

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useState } from "react";
+import { Box, Grid, Icon, TextField, Typography } from "@material-ui/core";
+
+import ProjectSummaryLabel from "./ProjectSummaryLabel";
+
+import {
+  PROJECT_UPDATE_INTERIM_ID,
+  PROJECT_CLEAR_INTERIM_ID,
+} from "../../../../queries/project";
+import { useMutation } from "@apollo/client";
+
+/**
+ * ProjectSummaryInterimID Component
+ * @param {Number} projectId - The id of the current project being viewed
+ * @param {Object} data - The data object from the GraphQL query
+ * @param {function} refetch - The refetch function from apollo
+ * @param {Object} classes - The shared style settings
+ * @param {function} snackbarHandle - The function to show the snackbar
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const ProjectSummaryInterimID = ({
+  projectId,
+  data,
+  refetch,
+  classes,
+  snackbarHandle,
+}) => {
+  // Instantiate Original Value
+  const [originalValue, setOriginalValue] = useState(
```

Comment:
_This could totally be a spin-off issue, and I don't think this blocks 🛳️-ing_

I know this is a copy an older component so I think there is a chance to simplify the logic a little. From what I'm seeing, the two pieces of state called `originalValue` and `interimId` can be combined. I did a quick test, and this seems to work but I could be missing a reason for the `useEffect`.

**The way this and the eCapris component work**
`props.data.moped_project[0].interim_project_id` -> initializes value of `originalValue` -> useEffect sets `interimId` when `originalValue` initializes or changes after through the UI

**Without the state called `originalValue` and the `useEffect`**
`props.data.moped_project[0].interim_project_id` -> initializes value of `interimId` and `setInterimId` updates `interimId` through the UI